backend/.env

```diff
@@ -0,0 +1,2 @@
+SECRET="secret_string"
+MONGODB_URI = "mongodb://localhost:27017/"
```

Comment:
The dotenv file is no longer needed in the repo. Please remove it from the commit.

Reply:
Removed.